Card/Masked.ts

```diff
@@ -0,0 +1,16 @@
+import { Expires } from "./Expires"
+
+export interface Masked {
+	masked: string
+	expires: Expires
+}
+export namespace Masked {
+	export function is(value: Masked | any): value is Masked {
+		return (
+			typeof value == "object" &&
+			typeof value.pan == "string" &&
+			Expires.is(value.expires) &&
+			typeof value.csc == "string"
```

Comment:
this function is (almost) identical to the one in Card, presumably there's an actual difference between the two? if you want to know if something is masked, surely you'd need to check the data not just its type